BAZProjectC1/Detail/View/DetailViewController.swift

```diff
@@ -21,15 +23,34 @@ class DetailViewController : UIViewController {
     }
     
     private func configureViewController() {
-        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        self.title = "Pelicula"
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil) 
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Buscar", style: .plain, target: self, action: #selector(searchMovieCV))
     }
     
     private func loadImage() {
+        lblTitle.text = objMov?.title
+        lblTitle.loadConfigurationFont(with: true)
         if let urlImage = objMov?.imageDetail {
             imgDetail.loadFrom(strUrl: "\(urlImage)")
         } else {
             imgDetail.image = UIImage.init(named: "placeHolder")
         }
+        txvDetail.text = objMov?.overview ?? "No existe descripcion de la pelicula que ingresaste"
+        txvDetail.loadConfigurationFont()
     }
     
+    @objc func searchMovieCV() {
```

Comment:
El nombre de la función no refleja claramente lo que hace, considera renombrarla, por ejemplo:
```suggestion
    @objc private func navigateToSearchMovieViewController() {
```

También considera si es necesario que esta función sea `internal`. ¿Podría ser privada?

Reply:
podría quedar como internal y / o privada por que solo se usa en este módulo